src/client/image_decoder/image_decoder.ts

```diff
@@ -33,33 +33,37 @@ export interface Image {
 }
 
 export class ImageDecoder {
-
   private atom: IAtom
 
   // These two functions are used to cache/destroy textures as they are created
   private cache: {
-    get(): Texture
-    destroy(): void
+    get(): Texture;
+    destroy(): void;
```

Comment:
Semicolons?! I thought we didn't have those in NUsight2. Doesn't tslint tell you to get rid of them?

Reply:
As statement terminators yes, but probably not in type definitions like here.